src/codegen/riscv64/assembler-riscv64.cc

```diff
@@ -2454,16 +2454,16 @@ int Assembler::li_estimate(int64_t imm, bool is_get_temp_reg) {
         continue;
       }
       // The first 1 seen
-      int32_t part;
+      //int32_t part;
```

Comment:
the style is bad. just delete the unused code, never comment out them. (the only case the code should live in comment is you have strong reason and write down as comments to explain)

Reply:
Ok, I learned that.